src/backend/aspen/database/models/sequences.py

```diff
@@ -213,7 +213,7 @@ class UploadedPathogenGenome(PathogenGenome):
     sample_id = Column(Integer, ForeignKey(Sample.id), unique=True, nullable=False)
     sample = relationship(  # type: ignore
         Sample,
-        backref=backref("uploaded_pathogen_genome", uselist=False),
+        backref=backref("uploaded_pathogen_genome", uselist=False, cascade="delete"),
```

Comment:
This is where we tell SqlAlchemy that if we delete a sample, we should also delete any UploadedPathogenGenome rows associated with it.